vcpkg/commands/test-features.md

```diff
@@ -0,0 +1,113 @@
+---
+title: vcpkg x-test-features
+description: Command line reference for the vcpkg test-features command. Builds all features of a package.
+ms.date: 03/07/2025
+---
+
+# vcpkg test-features
+
+## Synopsis
+
+```console
+vcpkg x-test-features [options] <package>...
+```
+
+## Description
+
+Tests all features of a package by installing them in manifest mode. This command is useful for testing that all features of a package are working correctly.
+The following tests are performed:
+- The core feature gets installed with default featured disabled
+- Every feature gets installed separately with default features disabled
+- All features combined gets installed
+
+Feature which are unsupported or marked as cascade in --ci-feature-baseline are skipped.
+
+## Example
+
+```console
+$ $ ./vcpkg x-test-features curl
+Computing 23 install plans...
+Computing all ABI hashes...
+Detecting compiler hash for triplet x64-linux...
+Compiler found: /usr/bin/c++
+Checking the binary cache...
+Feature Test [1/23] curl[core]:x64-linux
+Feature Test [2/23] curl[core,non-http]:x64-linux
+Feature Test [3/23] curl[core,tool]:x64-linux
+...
+```
+
+## Options
+
+All vcpkg commands support a set of [common options](common-options.md).
+
+### `--ci-feature-baseline`
+
+Path to the `ci.feature.baseline.txt` file. Used to skip ports and detect regressions.
+
+Each line in this file is an assignment of a port spec to a state. `#`s indicate comments, and
+blank lines are ignored.
+
+States:
+
+* **fail** - The port build is expected to fail in the CI system.  
+  This is not necessarily the same as if a port is expected to build on a developers machine
+  because it may fail due to the machine configuration.
+* **skip** - Do not build this port in the CI system.  
+  This is added to ports that may be flaky or conflict with other ports. We recommend adding a
+  comment for why a port is skipped so it can be removed when the issue is resolved.
+* **cascade** - The port depends on a port that fails or is excluded by a supports expression of a
+  dependency but the port itself states that it can be build. This is added to ports to detect
+  "hidden" not tested ports. 
+
+Additional States can be specified for specific features:
+* **feature-fails** - The listed feature do not build. They are therefore excluded from the
+  "combined" feature test because it would simply fail too.
+* **combination-fails** - The listed combination of features will fail to build.
+* **no-separate-feature-test** - The listed features are not tested in the "seperate" feature test.
+* **options** - The listed features are mutually exclusive and can not be selected together. The
```

Comment:
I think it's important to highlight that usually this behavior is prohibited in the vcpkg repo by linking the maintainer guide.